### PR TITLE
Palladium tube

### PR DESCRIPTION
**Pull request by ajnonaka**
Configuring a palladium tube with a current source.
Geometry can be figured with a series of parameters in the "my_constants" section.
IMG_0459

### CODE REVIEW

File: Examples/Tests/circuits/inputs_pd_electrode [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+warpx.use_filter = 0
+warpx.cfl = 0.9
+
+algo.em_solver_medium = macroscopic # vacuum/macroscopic
+
+algo.macroscopic_sigma_method = laxwendroff # laxwendroff or backwardeuler
+
+macroscopic.sigma_function(x,y,z) = "sigma_0 + (sigma_pd - sigma_0) * (x > -0.000001) * (x < 0.000001) * (z > -0.000001) * (z < 0.000001) + (sigma_si - sigma_0) * (z < -0.000001)"
+
+macroscopic.epsilon_function(x,y,z) = "epsilon_0 + (epsilon_r_pd - 1.) * epsilon_0 * (x > -0.000001) * (x < 0.000001) * (z > -0.000001) * (z < 0.000001) + (epsilon_r_si - 1.) * epsilon_0 * (z < -0.000001)"
```
Comment (RevathiJambunathan) [inline Examples/Tests/circuits/inputs_pd_electrode:29]: why epsilon_r_pd - 1 and epsilon_r_si - 1?

File: Examples/Tests/circuits/inputs_pd_electrode [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+macroscopic.mu_function(x,y,z) = "mu_0 + (mu_r_pd - 1.) * mu_0 * (x > -0.000001) * (x < 0.000001) * (z > -0.000001) * (z < 0.000001) + (mu_r_si - 1.) * mu_0 * (z < -0.000001)"
```
Comment (RevathiJambunathan) [inline Examples/Tests/circuits/inputs_pd_electrode:31]: same question here too

File: Examples/Tests/circuits/inputs_pd_electrode (hunk 1 above)
Comment (RevathiJambunathan) [reply Examples/Tests/circuits/inputs_pd_electrode:29]: oh I see it!

**Comment by ajnonaka** [comment]
Combining this PR with #43 so I am closing this.

**Pull request closed without merging**